udata/core/user/search.py

```diff
@@ -49,8 +49,8 @@ class UserSearch(ModelSearchAdapter):
         'views': Sort('metrics.views'),
         'created': Sort('created'),
     }
-    facets = {
-        'organization': ModelTermFacet('organizations', Organization),
+    aggregations = {
+        'user_organizations': ModelTermFacet('user_organizations', Organization),
```

Comment:
for coherence it should be `'user_organization': ModelTermFacet('user_organizations', Organization),` (singular)